chatterbox/src/Main/MainArea/Settings/user-settings-component.js

```diff
@@ -0,0 +1,84 @@
+import React, {Component} from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import Pencil from 'react-icons/lib/fa/pencil';
+import { Button } from 'material-ui';
+import './user-settings-component.css'
+import {
+    editUserNameRequest,
+    editUserEmailRequest,
+    editUserProfilePictureRequest,
+    saveChangesRequest
+} from '../../../modules/settings/edit-user-reducer.js'
+
+
+class UserSettings extends Component {
+
+    render() {
+        return (
+            <div className="user-details">
+                <div className="profile-picture">
+                    <div className="oval">
+                        <div className="oval-4">
+                            <img 
+                                src={this.props.user.profilePic} 
+                                alt="none"
+                                onClick={() => this.props.editUserProfilePictureRequest(this.props.user)}
+                            />
+                        </div>
+                    </div>
+                </div>
+
+                <div className="user-name">
+                    <h2 className="user-detail">{this.props.user.firstname} {this.props.user.lastname}</h2>
+                    <Pencil
+                        onClick={() => this.props.editUserNameRequest(this.props.user)} 
+                        className="edit"
+                    />
+                </div>
+                
+                <div className="user-email">
+                    <h3 className="user-detail">{this.props.user.email}</h3>
+                    <Pencil
+                        onClick={() => this.props.editUserEmailRequest(this.props.user)} 
+                        className="edit"
+                    />
+                </div>
+
+                <Button 
+                    onClick={() => this.props.saveChangesRequest(this.props.user)} 
+                    label="DONE"
+                    style={{

```

Comment:
added the style to .css

Reply:
i added a SubmitChanges object which contains the css for the <Button /> i could not connect  it to the external .css file